dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulationResult.java

```diff
@@ -6,35 +6,53 @@
  */
 package com.powsybl.dynamicsimulation;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 
-import com.powsybl.timeseries.RegularTimeSeriesIndex;
-import com.powsybl.timeseries.StringTimeSeries;
 import com.powsybl.timeseries.TimeSeries;
 
+import static com.powsybl.dynamicsimulation.DynamicSimulationResult.Status.SUCCEED;
+
 /**
  * @author Marcos de Miguel {@literal <demiguelm at aia.es>}
+ * @author Laurent Issertial {@literal <laurent.issertial at rte-france.com>}
  */
 public interface DynamicSimulationResult {
 
-    boolean isOk();
+    enum Status {
+        SUCCEED,
```

Comment:
Both value should have the same grammatically class:  `SUCCEEDED` and `FAILED`, or `SUCCEED` and `FAIL`, or `SUCCESS` and `FAILURE`. As we chose success and failure in the other APIs, we should stick with that.